app/views/pages/_banner.html.erb

```diff
@@ -4,13 +4,20 @@
       <div class="col-12 col-lg-6">
         <div class="banner-text-main position-relative" style="z-index: 9">
           <h1 class="mb-2">Master the Skills,<br>Master the LSAT</h1>
-          <p class="d-none d-md-block">LSAT Cracked is the only LSAT course that directly identifies the exact skills you're currently missing that an LSAT writer cares about. Stop wasting time repeating the same mistakes, and start unlocking your future career.</p>
-          <%= render "mail_leads/form", mail_lead: @mail_lead %>
-          <%= link_to "Test Your Skills With A Quiz", root_path(anchor: "intro-quiz"), class: "btn btn-primary mt-3" %>
+          <% if @current_user.nil? %>
```

Comment:
I'm surprised `@current_user` even exists. I'd rather change this to just `current_user` tho to make a more consistent use of devise's helper.